packages/client/src/clients/in-app-messages/types.ts

```diff
@@ -21,15 +24,21 @@ export interface InAppMessage<TContent = GenericData, TData = GenericData> {
   source: NotificationSource;
 }
 
-export interface InAppMessageResponse {
-  entries: InAppMessage[];
+export interface InAppMessageResponse<
```

Comment:
Not a blocker for this PR, but we should take a full pass later to get all naming things right/consistent (e.g. plural vs singular).

Reply:
Ack, got a todo for that